### PR TITLE
Update ScalaWebSockets.md

### PR DESCRIPTION
**Pull request by joesan**
# Pull Request Checklist

* [ ] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [ ] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [ ] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [ ] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [ ] Have you added copyright headers to new files?
* [ ] Have you checked that both Scala and Java APIs are updated?
* [ ] Have you updated the documentation for both Scala and Java sections?
* [ ] Have you added tests for any changed functionality?

# Helpful things

## Fixes

Fixes #xxxx

## Purpose

What does this PR do?

## Background Context

Why did you take this approach?

## References

Are there any relevant issues / PRs / mailing lists discussions?

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @joesan,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: documentation/manual/working/scalaGuide/main/async/ScalaWebSockets.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ And finally, we can use these in our WebSocket:
 
 Now in our actor, we will receive messages of type `InEvent`, and we can send messages of type `OutEvent`.
 
+You could also pass in request parameters to your WebSocket endpoint. This might be useful for cases where you want to apply some filtering on the stream of data that you get from the server. For example., your application might stream different types of events which are then identified by an EventID. You client could then pass in this EventID as a request parameter when it opens the WebSocket connection and in your WebSocket actor, you could filter all events that match the EventID.
```
Comment (ignasi35) [inline documentation/manual/working/scalaGuide/main/async/ScalaWebSockets.md]: I think there's a typo midway on the paragraph: "You client" --> "Your client"

**Comment by joesan** [comment]
@marcospereira Could you please help me how to do this?

**Comment by marcospereira** [comment]
Hi @joesan, I don't think Github has an automatic way of doing that. But you can easily do that using git.

```bash
# Clone your fork
git clone git@github.com:joesan/playframework.git
cd playframework
# checkout the branch for the first PR
git checkout patch-1
# cherry-pick the commit you made here
git cherry-pick 8b36eed
# Publish the cherry picked commit
git push origin patch-1
```
After that, your first PR (#8246) will have the changes made here.

Let me know if you have any problem with that. :-)

**Comment by marcospereira** [comment]
@joesan, I'm closing this since the work here should be merged with #8246.

Thanks!

**Pull request closed without merging**